test/integration/profiler_test.go

```diff
@@ -319,12 +343,26 @@ func TestAnyStackContains(t *testing.T) {
 	require.False(t, anyStackContains([][]string{{"a", "b"}}, []string{"a", "b", "c"}))
 }
 
+// profileDuration sets the profile runtime to a shorter time period
+// when running outside of CI. The logic for this is that very loaded
+// systems, such as GH actions might take a long time to spawn processes.
+// By increasing the runtime we reduce the chance of flaky test executions,
+// but we shouldn't have to pay this price during local dev.
+func profileDuration() time.Duration {
+	_, ok := os.LookupEnv("CI")
+	panic(ok)
```

Comment:
This will always panic regarless of the `ok` value. I assume this is not the intendent behaviour.

Reply:
Added this for testing, as it seemed to be running the wrong branch (see commit name)